bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -31,9 +31,9 @@ metadata:
     categories: Logging & Tracing
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2020-12-16T13:37:00+00:00"
+    createdAt: "2023-03-02T10:14:11Z"
```

Comment:
hm.. is this information displayed somewhere? Ignoring it makes sense, otherwise our diff test would fail. But at the same time, this field will be `2023-03-02T10:14:11Z` for ever.

Reply:
AFAIK, since the `operator-sdk` version `1.26`, the `createdAt` field is being changed.  Due to that, @iblancasa pointed to creating a workaround for it.

https://github.com/operator-framework/operator-sdk/issues/6285

Reply:
It seems it was not updated until now. So... it seems it is not really important.

Reply:
@yuriolisa yes, that thing helps to ignore the timestamp. But is it right to ignore this one? 
- If `createdAt` describes when the operator was first created, we should not update the timestamp.
- If the field describes when the shipped version was created, we should update the field with each release.

(My understanding is that a field like `createdAt` should never change and instead a field like `updatedAt` should be changed with each release.)

On [Operatorhub](https://operatorhub.io/operator/opentelemetry-operator) i found this:
![image](https://user-images.githubusercontent.com/10403402/223091866-cb33cbf7-d020-435f-b849-40d0f939492c.png)

Reply:
Since @frzifus reacted with `👎🏻`, I'll try to clarify my comment.

> It seems it was not updated until now. So... it seems it is not really important.

By this I meant: if it was expected to be updated, nobody paid attention to it. Sorry if I was not clear enough.
 
I think we should not update it but since it seems it is not clear what it means... I don't have a strong opinion. It can be:
* When the bundle was created
* When the operator was created